### PR TITLE
drop support for mime-type and file encoding.

### PR DESCRIPTION
**Pull request by diasbruno**

### CODE REVIEW

File: package.lisp [hunk 1]
```diff
@@ -5,9 +5,6 @@
   (:import-from #:serapeum
                 #:export-always
                 #:->)
-  (:local-nicknames (#:alex #:alexandria)
-                    (#:sera #:serapeum)
-                    (#:file-attributes #:org.shirakumo.file-attributes))
```
Comment (diasbruno) [inline package.lisp]: I think this is related to the uiop version shipped with SBCL (?). I doesn't recognize `local-nicknames`.

**Comment by diasbruno** [comment]
closes #1

File: package.lisp (hunk 1 above)
Comment (vindarel) [reply package.lisp]: What's your SBCL and ASDF version? I'd like to keep local nicknames if it isn't much to ask.

hint: https://github.com/lisp-tips/lisp-tips/issues/40

I don't remember when asdf 3.3.5 was released.

Comment (diasbruno) [reply package.lisp]: sbcl 2.3.9 and asdf 3.3.1. I rarely remember to update asdf.
I'll update and add back the nicknames.

Comment (diasbruno) [reply package.lisp]: Actually, sbcl 2.4.3. I've updated asdf, but still missing the package-local-nicknames.

Captura de tela 2024-05-06 094139

Comment (diasbruno) [reply package.lisp]: Oops. My bad. I changed to 'package-local-nicknames'. The correct was `local-nicknames`.

**Comment by diasbruno** [comment]
@vindarel It should be good now. 

What would be a good way to make extensions for this program?

File: package.lisp (hunk 1 above)
Comment (vindarel) [reply package.lisp]: great, thanks.

**Comment by vindarel** [comment]
great, thanks.

I think the easiest way will be to define additional systems in the same .asd file.

**Pull request merged**